File: src/zasz/me/disorganizerfonts/DisorganizerFonts.java
package zasz.me.disorganizerfonts;

import java.awt.*;
import java.io.IOException;

public enum DisorganizerFonts
{

    AlphaFridgeMagnets,
    AlphaFridgeMagnetsAllCaps,
    Gnuolane,
    KenyanCoffee,
    Sexsmith,
    Steelfish,
    VenusRising;

    public static Font GetFont(DisorganizerFonts FontName) throws IOException, FontFormatException
    {
       return Font.createFont(
               Font.TRUETYPE_FONT,
               DisorganizerFonts.class.getResourceAsStream(FontName.name() + ".ttf")
       );
    }
}

File: src/zasz/me/disorganizerfonts/DisorganizerFonts.scala
package zasz.me.disorganizerfonts

import java.awt._
import javax.servlet.ServletContext

object DisorganizerFonts extends Enumeration
{
  type DisorganizerFonts = Value

  def getFont(fontname: DisorganizerFonts, servletContext: ServletContext): Font =
  {
    Font.createFont(Font.TRUETYPE_FONT, getClass.getResourceAsStream(fontname.toString + ".ttf"))
  }

  final val AlphaFridgeMagnets = Value("AlphaFridgeMagnets")
  final val AlphaFridgeMagnetsAllCaps = Value("AlphaFridgeMagnetsAllCaps")
  final val Gnuolane = Value("Gnuolane")
  final val KenyanCoffee = Value("KenyanCoffee")
  final val Sexsmith = Value("Sexsmith")
  final val Steelfish = Value("Steelfish")
  final val VenusRising = Value("VenusRising")
}